src/Data/Hashabler/Internal.hs

```diff
@@ -616,7 +618,8 @@ hash32BigNatByteArrayBytes h numLimbs ba =
 instance Hashable Natural where
     {-# INLINE hash #-}
     hash h nat = case nat of
-# if defined (MIN_VERSION_integer_gmp) && MIN_VERSION_integer_gmp(1,0,0)
+# if defined (MIN_VERSION_integer_gmp)
+#   if MIN_VERSION_integer_gmp(1,0,0)
```

Comment:
I had to separate these 2 checks. Combining with `&&` results in an error:

```cpp
if defined X && X(1,0,0)
```

Consider the case where X is indeed not defined:

```cpp
if defined && (1,0,0)
```

No you will get some error complaining about terniary operators.